test/test_auth_service.py

```
@@ -0,0 +1,130 @@
+        }
+
+        result = auth_service.get_or_create_user(kakao_user_info)
+
+        assert result.kakao_id == "12349"
+        assert result.nickname == DEFAULT_NICKNAME
+        assert result.profile_image == DEFAULT_PROFILE_IMAGE
+        assert result.email == "no_properties@example.com"
+
+    def test_get_or_create_existing_user(self, auth_service, db_session):
+        """기존 사용자가 있는 경우 테스트"""
+        from .fixtures import UserFixture
```

Ruff (0.12.2)

Redefinition of unused `UserFixture` from line 4

Remove definition: `UserFixture`

(F811)